components/script/script_thread.rs

```diff
@@ -18,6 +18,7 @@
 //! loop.
 
 use bluetooth_traits::BluetoothRequest;
+use core::borrow::BorrowMut;
```

Comment:
It was giving me an error when I used the borrow_mut() in the add_mutation_observer, now that I have change it to DOMRefCell, I guess this won't be needed anymore. I'll remove it.